views/score.erb

```diff
@@ -0,0 +1,28 @@
+<!--  Form to score a single word -->
+<form action="" method="post" accept-charset="utf-8">
+  <label for="word"> Your Word</label>
+  <input type="text" name="word" value="" id="word">
+  <input type="submit" value="Score my Word">
+</form>
+
+
+<img src="/score.png" width = "400" alt= "scrabble score">
+
+<<<<<<< HEAD
```

Comment:
Looks like you have a merge conflict it this file!! EEK! 

You probably got a message in your commit/pull process that should have been resolved before you committed in order to ensure you have functional code. See [this link](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging#Basic-Merge-Conflicts) for more information